src/randoop/NoExceptionCheck.java

```diff
@@ -3,8 +3,10 @@
 import randoop.sequence.Execution;
 
 /**
- * Checks that no exception is thrown after a given statement
- * in a sequence.
+ * Checks that an exception is not thrown by a given statement in a sequence. 
+ * This is meant for an error-revealing check where, in fact, an exception
+ * was observed at the statement, and this check is meant to represent that
+ * the throws is actually an expected failure.
```

Comment:
I've revised this.

Incidentally, there is no actual check for exceptions in error-revealing tests. Only a comment is output. If the test isn't flaky, the exception will be thrown when the error-revealing test is run.

I'm thinking we should revisit what error-revealing tests do, but not in this PR